<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1259</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>811</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>390</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>256</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;解密：9=&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>输入答案</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>481</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>原神玩家列表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>330</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>添加原神玩家</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>360</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>删除原神玩家</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>110</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>随机</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>苏师傅</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>酥师傅</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>颜黎</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>110</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NOIP</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>140</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>随机</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>苏师傅</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>酥师傅</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>颜黎</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>170</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>随机</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>苏师傅</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>酥师傅</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>颜黎</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>140</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NOI</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>170</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CSP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>启动</string>
    </property>
   </widget>
   <widget class="mediaplayer" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>280</y>
      <width>401</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="browser" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>20</y>
      <width>401</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>朗读文本</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1259</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="action"/>
    <addaction name="actiontiger"/>
    <addaction name="actionbrowser"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>打印</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>撤销</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>重做</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>俄罗斯</string>
   </property>
  </action>
  <action name="actiontiger">
   <property name="text">
    <string>老虎</string>
   </property>
  </action>
  <action name="actionbrowser">
   <property name="text">
    <string>浏览器</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mediaplayer</class>
   <extends>QWidget</extends>
   <header location="global">mediaplayer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>browser</class>
   <extends>QWidget</extends>
   <header location="global">browser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
